tests/plugins/editorplaceholder/editorplaceholder.js

```diff
@@ -205,5 +233,93 @@
 		} );
 	};
 
+	// (#4253)
+	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin' ] = function() {
+		addErrorAsserts( this );
+
+		bender.editorBot.create( {
+			name: 'bbcodeFailTest',
+			config: {
+				extraPlugins: 'bbcode',
+				editorplaceholder: 'any',
+				fullPage: true,
+				on: {
+					instanceReady: function() {
+						resume( function() {
+							assert.isFalse( this.consoleErrorStub.called, 'There should be no errors during initialization' );
```

Comment:
Didn't notice before but I wonder why this assert is here and not in `bender.editorBot.create` callback?

Reply:
This comment is not relevant anymore see https://github.com/ckeditor/ckeditor4/pull/4370#discussion_r526898877.

Reply:
>The correct way would be to spy/stub console error with sinon to see if no errors where thrown - if error is thrown use assert.fail( ... ) to fail the test and if editor fires instanceReady event test should be marked as passing with assert.pass( ... ).

Maybe I took your comment too literally.